cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1528,13 +1528,12 @@ function Configure-LoggingAgent {
   if (-not (ShouldWrite-File $fluentd_config_file)) {
```

Comment:
Instead of adding the else block, can you just remove this `if (ShouldWrite-File ...)` block entirely? Then I'm guessing that all the code below can stay the same and the file will be written / overwritten every time.